newIDE/app/src/VariablesList/index.js

```diff
@@ -367,6 +383,16 @@ export default class VariablesList extends React.Component<Props, State> {
     const { variablesContainer, inheritedVariablesContainer } = this.props;
     if (!variablesContainer) return null;
 
+    if (this._allVariableNamesSet === null) {
+      const allVariableNames = this.props.onComputeAllVariableNames();
+      this._allVariableNamesSet = new Set<string>(allVariableNames);
+      this._undefinedVariableNames = new Set<string>(
```

Comment:
Not sure if this will work with structure having children: if we have an event using the scene variable called "MyVariable.MyChild" for example?